src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/metrikker/BedriftsmetrikkerController.java

```diff
@@ -0,0 +1,39 @@
+package no.nav.arbeidsgiver.sykefravarsstatistikk.api.metrikker;
+
+import no.nav.arbeidsgiver.sykefravarsstatistikk.api.domene.Orgnr;
+import no.nav.arbeidsgiver.sykefravarsstatistikk.api.enhetsregisteret.EnhetsregisteretClient;
+import no.nav.arbeidsgiver.sykefravarsstatistikk.api.enhetsregisteret.Underenhet;
+import no.nav.security.oidc.api.Unprotected;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+import java.math.BigDecimal;
+
+@Unprotected
```

Comment:
Jeg var jo pådriver for at denne skulle være Unprotected. Men jeg innser at det kanskje er like greit at den er Protected likevel. Det gir jo bare et ekstra sikkerhetslag, og brukere som kaller dette endepunktet på tiltenkt måte er jo innlogget. Hva synes du?

Men vi kan fremdeles droppe tilgangskontroll

Reply:
Det er jo et fint sikkerhetsprinsipp - blokkér alt by default, og åpne opp for spesifikke tilfeller